README.md

```diff
@@ -1,19 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# CRYPTO-CON 2022
 
-> Description the project.
+> Here I made a web page for a non existing crypto-con. 
+
+[Project Mockup](mockup-readme.jpg)
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+-CSS
+-HTML
+-JavaScript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](#) [Updating Link when merge with main branch]
```

Comment:
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:


- A description of the project.
- A demo of the project features (different pages and different screen sizes).
- You should also highlight some interesting piece of code or something you built that you are very proud of.
- For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture.


Add the video link to your pull request description.